src/en/proposals/pjb-multi-character-slot.md

```diff
@@ -0,0 +1,60 @@
+# Multiple Character Slots
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| PJB3005 | :x: No | TBD |
+
+## Background
+
+Currently, a player can have one character slot selected. This controls their character's appearance, name, job preferences, and antag preferences. This is relatively restrictive, as people may want to have different settings per job. For example:
+
+* Some people prefer having different characters per job or department, for roleplay reasons.
+* The work on [loadouts](https://github.com/space-wizards/space-station-14/pull/25715) makes it so that loadouts are assigned per-job, which is kind of clunky.
+* Some "jobs" like clown or borg shouldn't use your normal character name. In SS14 this is currently suffering, in SS13 there's either a popup box when you spawn to pick your name OR multiple extra fields in the character info for "character name as clown" and such. Both suck.
+
+## Proposal
+
+It should be possible to have multiple character slots active at once. When selecting jobs, a job is picked for all your characters at once, after which the actual character slot is picked.
+
+All the regular properties of a character slot apply. You can have separate names, appearances, and with the loadouts PR separate loadouts too.
+
+As an example: if you have two character slots, one engineer, one medical doctor, the game will try to give you a job for either. The actual character slot then gets picked based on what job you got.
+
+### UI
+
+Most of the UI would remain mostly the same.
+
+The character customization screen currently lists all your characters in a column on the left. With this change, it'd create two separate headings: one for "active" and one for "inactive". There would be buttons to activate/deactivate character slots.
+
+The lobby currently shows your active character in the menu. With this change, that would instead show all your character slots lined up horizontally. This is also nice because if you do take advantage of multiple character slots this allows you to easily tell "I have engie, med and sec selected".
+
+We will probably need a "duplicate" button to allow you to clone a character slot, if you want to make a character slot with the same appearance but a different job easily.
+
+### Job/character selection
+
+The game would run job selection first, with your apparent job preferences selected as the union of the preferences of all your characters. Then when a job has been picked for you, your character slot is picked based on what jobs are selected on your characters. If more than one character has the job selected, it'd just be picked randomely between them.
```

Comment:
This is very complicated and would be confusing un-explained behavior to a player. Job preferences should not be tracked on a per character basis, they are something that a player has. Job preferences should reflect those of the player themselves since they are effectively a consent system for the type of gameplay that a player wishes to engage in during that session. A better solution would be to have a system for managing Job Preference presets, aka the ability to save/select saved preferences.